federation/manifests/federation-apiserver-deployment.yaml

```diff
@@ -23,7 +23,6 @@ spec:
         - --etcd-servers=http://localhost:2379
         - --service-cluster-ip-range={{.FEDERATION_SERVICE_CIDR}}
         - --secure-port=443
-        - --advertise-address={{.FEDERATION_API_HOST}}
```

Comment:
What existing behavior are we breaking? As far as I know, there is no functionality in the federation control plane which is affected by the `--advertise-address` simply defaulting to the bind address. If there were, we should fix it so it will work on all platforms, rather than just GCE/local dev (rather than make this change here).